cmd/wallet-sdk-gomobile/docs/usage.md

```diff
@@ -364,7 +366,7 @@ let didResolver = DidNewResolver("", nil)
 let didCreator = DidNewCreatorWithKeyWriter(kms, nil)
 let didDocResolution = didCreator.create("key", ApiCreateDIDOpts()) // Create a did:key doc
 let activityLogger = MemNewActivityLogger() // Optional, but useful for tracking credential activities
-let cfg =  Openid4ciClientConfig(didDocResolution.id, clientID: "ClientID", didRes: didResolver, activityLogger: activityLogger)
+let cfg =  Openid4ciClientConfig(didDocResolution.id, clientID: "ClientID", didRes: didResolver, activityLogger: activityLogger, disableVCProofChecks: false)
```

Comment:
is proof check enabled by default, or should it be enabled by "disableVCProofChecks: false"?

Reply:
I've removed it from the constructor and instead exposed the option via a `DisableVCProofChecks` method that should hopefully make it clearer.